### ISSUE TITLE
When the network is lost the message "Reading image file ..."  is printed but the image is not loaded

### ISSUE THREAD

**Issue by r-shahpasand** [marcglasberg/network_to_file_image#17]
When network is lost the message "Reading image file ..."  is printed but the image doesn't load. With a network connection, the image is read from file and displayed properly.

**Comment by marcglasberg** [comment]
You mean, without internet it doesn't load from file? It tries to load from file before trying the internet, so that's strange. Or I didn't understand what you mean. Could you give me more detail?

**Comment by r-shahpasand** [comment]
Thanks for your reply. Yes you get it right. As you said it read saved file but without internet it doesn't show the image, while with the Internet connection this problem doesn't exist. I'm using the code you provided in example section.

**Comment by marcglasberg** [comment]
Ok, that's really strange. I'll have a look at it.

**Comment by marcglasberg** [comment]
Are you using Flutter 1.20? Could you please post your `flutter doctor`?

**Comment by r-shahpasand** [comment]
G:\Projects\Learn>flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, v1.17.3, on Microsoft Windows [Version 10.0.18362.720], locale en-US)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[√] Android Studio (version 4.0)
[!] VS Code (version 1.45.1)
    X Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
[√] Connected device (1 available)

! Doctor found issues in 1 category.

**Comment by marcglasberg** [comment]
I have upgraded the package to Flutter 1.20. There were indeed some changes I needed to make in the `ImageForCanvas` class, but not for the `NetworkToFileImage` class. In any case you are not using Flutter 1.20.

I can't reproduce the problem you describe. What I did was this:

1) Run once the example from https://github.com/marcglasberg/network_to_file_image/blob/master/example/lib/main.dart
  This would download the image and save it to file.

2) Turn off the internet, by putting the emulator into airplane mode.

3) Run again the example. The image is displayed.
